src/compute.rs

```diff
@@ -52,6 +89,42 @@ pub fn binding_compressed(id: &Scalar, B: &[(Compressed, Compressed)], msg: &[u8
         .expect("FATAL: DST is less than 256 bytes so operation should not fail")
 }
 
+/// Compute a binding value from the party ID, public nonces, and signed message using default expansion.
```

Comment:
We should probably explain here, and in the expansion type enum variant docs, where these binding algorithms are coming from (link to the paper and the RFC) and why someone might prefer one over the other.

Reply:
I'd probably add the link to the paper or RFC that explains the algorithm that this function is implementing. Same thing for the XMD function too. That stuff is always handy near the function as well.